src/app/delegation_backend/src/cmd/delegation_backend/main_bpu.go

```diff
@@ -25,20 +27,24 @@ func main() {
 
 	ctx := context.Background()
 
+	appCfg := LoadEnv(log)
+
+	awsCfg, err := config.LoadDefaultConfig(ctx, config.WithRegion(appCfg.Aws.Region))
+	if err != nil {
+		log.Fatalf("Error loading AWS configuration: %v", err)
```

Comment:
Shouldn't we exit here as in previous version?

Reply:
The Fatal functions from the log package call os.Exit(1) after writing the log message, so we are in fact exiting.